dev/com.ibm.ws.jaxrs.2.0_fat/fat/src/com/ibm/ws/jaxrs20/fat/ExceptionMappingWithOTTest.java

```diff
@@ -44,7 +43,7 @@
  * the MP OT mapper.
  */
 @RunWith(FATRunner.class)
-@SkipForRepeat(JakartaEE10Action.ID) //MP Open Tracing replace with MP Telemetry so this test is not valid beyond EE9
+@SkipForRepeat({SkipForRepeat.EE9_FEATURES, SkipForRepeat.EE10_FEATURES, SkipForRepeat.EE11_FEATURES}) //MP Open Tracing replace with MP Telemetry so this test is not valid beyond EE9
```

Comment:
```suggestion
@SkipForRepeat({SkipForRepeat.EE10_FEATURES, SkipForRepeat.EE11_FEATURES}) //MP Open Tracing replace with MP Telemetry so this test is not valid beyond EE9
```
EE9 repeat should still be done based off of the old settings.